secret-squirrel.js

```diff
@@ -6,7 +6,8 @@ module.exports = {
 	strings: {
 		deny: [],
 		denyOverrides: [
-			'6da31a37-691f-4908-896f-2829ebe2309e'
+			'isLiveBlogV1OrPackage', // src/lib/get-topper-settings.js:18|234
```

Comment:
This variable name was flagged by Secret Squirrel: 
```
file: src/lib/get-topper-settings.js
line number: 234
full line: 	} else if (isLiveBlogV1OrPackage(content)) {
deny match: isLiveBlogV1OrPackage
deny rule: [A-Za-z0-9]{21}

Error: denied strings found
```
I've let the Platforms team know as I don't think this is the intended behaviour.